### PR TITLE
Add tracker to data collection

### PR DESCRIPTION
**Pull request by iakov-aws**
*Issue #, if available:*

*Description of changes:*
Add tracker and testing capabilities to data collection lab

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by awssteph** [comment]
Please update the ODC FAQ how to update athena link and FAQ commands

File: static/Cost/300_Optimization_Data_Collection/Code/source/s3_files_migration.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+"""
+Moving s3 objects from old structure to the new one
+
+Usage:
+    python3 {prog} <payer_id> <ODC_bucket>
```
Comment (awssteph) [inline static/Cost/300_Optimization_Data_Collection/Code/source/s3_files_migration.py:5]: The newest version has multiple payers data in the folders. Can we take this <payer_id> out and read it from the current object in S3 pleasae

Comment (iakov-aws) [reply static/Cost/300_Optimization_Data_Collection/Code/source/s3_files_migration.py]: 1) if it is a new structure (v2) with payers, then the migration will not change anything
2) I added a code to read payer account from org API, so we do not need customer to enter it. It is only needed if we have migration from v1.

**Comment by iakov-aws** [comment]
> Please update the ODC FAQ how to update athena link and FAQ commands

@awssteph  done

**Review by awssteph — APPROVED** [review]

**Pull request merged**